### PR TITLE
Support an external list of terms in term list operators

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: container-search/src/main/java/com/yahoo/search/yql/ParameterListParser.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        while (s.peek() != '}') {
+            String key;
+            if (s.passOptional('\'')) {
+                key = s.stringTo(s.position('\''));
+                s.pass('\'');
+            }
+            else if (s.passOptional('"')) {
+                key = s.stringTo(s.position('"'));
+                s.pass('"');
+            }
```
Comment (jonmv) [inline container-search/src/main/java/com/yahoo/search/yql/ParameterListParser.java:51]: Keys like `foo'bar"baz:bax` are not possible then, but that's OK.

**Review by jonmv — APPROVED** [review]
Great. Would we want to allow this for other similar term lists as well?

**Comment by bratseth** [comment]
Like what? This covers WeakAnd, WAND and DotProduct

**Comment by jonmv** [comment]
Right, they all use the weighted set, of course.

**Pull request merged**